tekton/pipeline_run.go

```diff
@@ -69,6 +70,20 @@ func (r *ReleasePipelineRun) AsPipelineRun() *tektonv1beta1.PipelineRun {
 	return &r.PipelineRun
 }
 
+// WithEnterpriseContractConfigMap adds a param providing the verify ec task bundle to the release PipelineRun.
+func (r *ReleasePipelineRun) WithEnterpriseContractConfigMap(cm *corev1.ConfigMap) *ReleasePipelineRun {
```

Comment:
Rename to `configMap`. We are not in a letters shortage.

Reply:
done